File: output/ChatGPT/avatar/Java/Python/codeforces_327_A.py
def check_no(brr):
    val = 0
    for i in brr:
        if i == 1:
            val += 1
    return val

if __name__ == "__main__":
    n = int(input())
    arr = list(map(int, input().split()))
    max_val = float("-inf")
    
    for i in range(n):
        for j in range(i, n):
            x = 0
            brr = arr.copy()
            for k in range(i, j+1):
                if brr[k] == 0:
                    brr[k] = 1
                else:
                    brr[k] = 0
            count = check_no(brr)
            if count > max_val:
                max_val = count
    
    if n == 1:
        if arr[0] == 1:
            print(0)
        else:
            print(1)
    else:
        print(max_val)

File: output/ChatGPT/avatar/Java/C++/codeforces_327_A.cpp
#include <iostream>
using namespace std;

int checkNo(int arr[], int n) {
    int val = 0;
    for (int i = 0; i < n; i++) {
        if (arr[i] == 1) {
            val++;
        }
    }
    return val;
}

int main() {
    int n;
    cin >> n;
    int arr[n];
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }
    int max = INT_MIN;
    for (int i = 0; i < n; i++) {
        for (int j = i; j < n; j++) {
            int x = 0;
            int brr[n];
            for (int k = 0; k < n; k++) {
                brr[x++] = arr[k];
            }
            for (int k = i; k <= j; k++) {
                if (brr[k] == 0) {
                    brr[k] = 1;
                } else {
                    brr[k] = 0;
                }
            }
            int count = checkNo(brr, n);
            if (count > max) {
                max = count;
            }
        }
    }
    if (n == 1) {
        if (arr[0] == 1) {
            cout << 0 << endl;
        } else {
            cout << 1 << endl;
        }
    } else {
        cout << max << endl;
    }
    return 0;
}